azurerm/data_source_eventhub_namespace_test.go

```diff
@@ -83,7 +84,9 @@ resource "azurerm_eventhub_namespace" "test" {
   resource_group_name      = "${azurerm_resource_group.test.name}"
   sku                      = "Standard"
   capacity                 = "2"
-  auto_inflate_enabled     = true
+	auto_inflate_enabled     = true
```

Comment:
Looks like line 86 and 87 are duplicates.

Reply:
A change like this usually means the line break when from `\n` <--> `\n\r`

Reply:
Sorry, I don't understand. Are you saying lines 87 and 88 are not duplicates, but appear so due to github formatting? It looks to me like they are two green + lines with the same content.

Reply:
Oh my apologies, you are definitely correct. Line 88 didn't appear in the conversation view.